packages/core-modules/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@synthetixio/core-modules",
+  "version": "1.0.0",
+  "description": "",
+  "files": [
+    "contracts/*",
+    "!contracts/Proxy.sol",
```

Comment:
This `Proxy` implementation is necessary for tests purposes, but it shouldn't be added to the npm package.